Formula/cockroach@22.2.rb

```diff
@@ -4,15 +4,15 @@
 class CockroachAT222 < Formula
   desc "Distributed SQL database"
   homepage "https://www.cockroachlabs.com"
-  version "22.2.17"
+  version "22.2.19"
```

Comment:
confirming that this is what we want!
(i forgot to brew update for 22.2.18 - thanks for bumping to 22.2.19!)